### ISSUE TITLE
Why yapDatabaseRelationshipEdges can't called?

### ISSUE THREAD

**Issue by bimawa** [yapstudios/YapDatabase#424]
i have iterface `@interface MessageInfoDBModel : BaseObject <YapDatabaseRelationshipNode>`
but in implementation this method `- (NSArray *)yapDatabaseRelationshipEdges` not called, when i do save or change record. What i do wrong?

**Comment by bimawa** [comment]
solved with manualy:

```
[self.backgroundConnection readWriteWithBlock:^(YapDatabaseReadWriteTransaction *transaction) {
      [transaction removeObjectForKey:chatInfoDBModel.identifier.stringValue inCollection:self.currentCollectionName];
      NSMutableArray *arrayOfKeysForDelete = [NSMutableArray new];
      [transaction enumerateKeysInCollection:[HMIDataBaseNamesStorageDBService messagesInfoCollectionNameChatId:chatInfoDBModel.identifier] usingBlock:^(NSString *key, BOOL *stop) {
        [arrayOfKeysForDelete addObject:key];
      }];
      [transaction removeObjectsForKeys:arrayOfKeysForDelete inCollection:[HMIDataBaseNamesStorageDBService messagesInfoCollectionNameChatId:chatInfoDBModel.identifier]];
    }];
```

**Comment by robbiehanson** [comment]
Did you setup the YapDatabaseRelationship extension within your app ?

It's mentioned at the very end of the wiki article:
https://github.com/yapstudios/YapDatabase/wiki/Relationships

(Should probably be at the beginning huh...)

```
[database asyncRegisterExtension:[YapDatabaseRelationship new] withName:@"relationships" completionBlock:nil];
```

**Comment by bimawa** [comment]
@robbiehanson yes you are right! And bug fix from #399 too helped.
